reconciletags/reconciletags_e2e_test.py

```diff
@@ -96,9 +97,32 @@ def testTagReconciler(self):
 
         # now try with a fake digest
         self._TEST_JSON['projects'][0]['images'][0]['digest'] = 'fakedigest'
-        with self.assertRaises(subprocess.CalledProcessError):
+        with self.assertRaises(AssertionError):
             self.r.reconcile_tags(self._TEST_JSON, False)
 
+    def testParFile(self):
+        subprocess.call(['bazel', 'build', 'reconciletags:reconciletags.par'])
```

Comment:
Could we make this whole test file run in bazel test? Then you won't need to do the build in here or hardcode the bazel-bin path below.